e2e/page-objects/profilePage.ts

```diff
@@ -0,0 +1,48 @@
+import { expect } from '@playwright/test';
+import { HelperBase } from './helperBase';
+
+export class ProfilePage extends HelperBase {
+  /**
+   * By default, this method checks that at profile page there is a logout button. Also checks that at least one element with "Profile" text is present. After that it compares the email got with email passed to it. But also it can check that no 'Profile' texts and logout buttons are present
+   * @param email expected email to what value should be equal to
+   * @param isVisible true to check profile presence, false - profile absence
+   */
+
+  async checkProfilePageForLogoutBtnAndProfileTitleAndCheckEmailIfNeeded(
+    email: string = 'email',
+    isVisible: boolean = true,
```

Comment:
It better to simplify the method, with removing isVisible flag

```ts
  async checkProfilePageForLogoutBtnAndProfileTitleAndCheckEmail(email: string) {
    const logoutBtn = this.page.getByRole('button', { name: 'Logout' });
    const profileTextElements = this.page.getByText('Profile').nth(1);

    await expect(logoutBtn).toBeVisible();
    await expect(profileTextElements).toBeVisible();
    const actualEmail = await this.getProfileEmailValueAndCheckFieldIsDisabled();
    expect(actualEmail).toEqual(email);
  }
  ```

Reply:
done